### PR TITLE
docs: replace imdb references and remove browser SDK docs page

### PR DESCRIPTION
**Pull request by minaelee**
- delete typescript browser SDK page
- add supported runtimes to TypeScript Node SDK page
- replace imdb references throughout documentation with petstore/generic
  - remaining imdb references in Fern's Documentation are:
    - docs for `fern init` as the command still produces imdb.yml
    - docs for fastapi boilerplate generator, as it references fastapi-starter heavily, which uses imdb

### CODE REVIEW

File: fern/docs/pages/fern-sdks/other-generators/server-side/fastapi.mdx [hunk 1]
```diff
@@ -1,8 +1,3 @@
----
```
Comment (dannysheridan) [inline fern/docs/pages/fern-sdks/other-generators/server-side/fastapi.mdx:1]: @minaelee why remove this?

File: fern/docs/pages/fern-sdks/sdk-generators/java-sdk.mdx [hunk 2]
```diff
@@ -56,12 +56,12 @@ groups:
         version: 0.6.1
         output:
           location: maven
-          # replace with your coordinate
+          # replace YOUR_API with your coordinate
```
Comment (dannysheridan) [inline fern/docs/pages/fern-sdks/sdk-generators/java-sdk.mdx:59]: where is YOUR_API referenced?

File: fern/docs/pages/fern-sdks/sdk-generators/node-sdk.mdx [hunk 3]
```diff
@@ -1,79 +1,90 @@
...
 
-<CodeBlock title="generators.yml">
+<CodeBlock title='generators.yml'>
 ```yaml
 groups:
   internal:
     generators:
       - name: fernapi/fern-typescript-node-sdk
-        version: 0.9.5
+        version: 0.x.x # replace with current or desired version
```
Comment (dannysheridan) [inline fern/docs/pages/fern-sdks/sdk-generators/node-sdk.mdx:49]: can we simplify to "declare your desired version"?

**Comment by dannysheridan** [comment]
@minaelee left comments in-line. Once resolved, good to merge.

The smaller the PR, the easier for me to code review. Would recommend splitting up each of the 3 bullet points of the PR summary into their own PR in the future.

**Comment by minaelee** [comment]
@dannysheridan should all be resolved, thanks!

**Pull request merged**